presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -1383,7 +1383,7 @@ private void testInsertPartitionedTable(Session session, HiveStorageFormat stora
 
         assertQuery(
                 session,
-                "SELECT * FROM " + partitionsTable,
+                "SELECT SHIP_PRIORITY, ORDER_STATUS FROM " + partitionsTable,
```

Comment:
Why do you need this change and other similar ones below?

Reply:
Otherwise the assertion fails because one query has too many columns in the output (the actual query has the $partition_stats column while the expected does not).

Reply:
Then I think it makes sense to fix the expected to reflect the new thing.

Reply:
Adding the "$partition_stats" column to the expected on those tests is kind of weird, since the expected query is querying from a different table with the same schema and data... We'd have to hardcode the JSON for each row, which is a) annoying and b) dependent on the formatting/whitespace of the JSON (tests fail if JSON formatting changes or the JSON object structure changes). I think we should just make the column hidden if it's a problem to change the actual of the tests to not include the "$partition_stats" column.

Reply:
The spirit of this test seems to check for the schema of the $partitions table which is supposed to have just the partition columns earlier but now we have more. So your fix is defeating the purpose of the test. I say change it to test the result columns using MaterializedResult to make sure it has 3 columns and two of them are identical to the result you get from the expected query.